### PR TITLE
FUSETOOLS-3519: Fix UI Test 'DataTransformationDeploymentTest' and convert it into master branch

### PR DESCRIPTION
**Pull request by pospisilf**
# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by pospisilf** [comment]
Should be improved.

**Comment by pospisilf** [comment]
rebased

**Comment by pospisilf** [comment]
testPR

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/DataTransformationDefaultTest.java [hunk 1]
```diff
@@ -194,8 +222,10 @@ protected void createMapping() {
 	protected void addDataTransformation() {
 		CamelEditor editor = new CamelEditor();
 		editor.addCamelComponent("Data Transformation", "SetHeader _setHeader1");
-		new WaitUntil(new ShellIsAvailable("New Fuse Transformation"), TimePeriod.VERY_LONG);
+		new WaitUntil(new ShellIsAvailable("New Fuse Transformation"), TimePeriod.getCustom(600));
+//		new WaitUntil(new ShellIsActive("New Fuse Transformation"), TimePeriod.getCustom(600));
```
Comment (apupier) [inline uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/DataTransformationDefaultTest.java:226]: commented out lien can be removed

**Review by apupier — CHANGES_REQUESTED** [review]

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/DataTransformationDefaultTest.java (hunk 1 above)
Comment (pospisilf) [reply uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/DataTransformationDefaultTest.java]: Fixed, thanks for noticing.

**Comment by pospisilf** [comment]
Skipping tests affected by opened JIRA issues can be handled better. Will be redone.

**Pull request closed without merging**